src/style.css

```diff
@@ -1,3 +1,91 @@
+*,
+*::before,
+*::after {
+	padding: 0;
+	margin: 0;
+	border: 0;
+  box-sizing: border-box;
+}
+
+li {
+	list-style: none;
+}
+
+a {
+	text-decoration: none;
+	color: inherit;
+}
+
+img {
+	vertical-align: top;
+}
+
+html {
+  --header-color: #00ACDC;
+}
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header__container {
+  max-width: 1200px;
```

Comment:
Actually we can remove this `margin: 0 auto` and `max-width` to allow our header to stretch on large screens, but to keep it from being smaller than 1024px, let's add `min-width: 1024px`